sdk/core/core-amqp/src/auth/sas.ts

```diff
@@ -109,20 +104,17 @@ export class SasTokenProvider implements TokenProvider {
     const sig = encodeURIComponent(shaObj.getHMAC("B64"));
     return {
       token: `SharedAccessSignature sr=${audience}&sig=${sig}&se=${expiry}&skn=${keyName}`,
-      tokenType: TokenType.CbsTokenTypeSas,
-      expiry: expiry
+      expiresOnTimestamp: expiry
```

Comment:
Is the unit of measurement the same for `expiresOnTimestamp` as it was for `expiry`?